content/20.results.md

```diff
@@ -7,130 +7,97 @@ Keynote speakers for RECOMB were available for all years from 1997--2019.
 
-We directly compared honoree and author results from 1993 to 2019 for the predicted proportion of white, Asian, and other categories (see [notebook](https://greenelab.github.io/iscb-diversity/13.us-race-analysis.html#what-about-p-values) and Fig. {@fig:us_racial_makeup}E).
-We found that, over the years, white honorees have been significantly overrepresented (t~218~ = 14.8, _p_ < 10^-16^) and Asian honorees have been significantly underrepresented (t~236~ = -18.8, _p_ < 10^-16^).
-A Welch two-sample t-test did not reveal any statistically significant difference in the mean probability of ISCB speakers predicted to be in Other categories compared to authors (t~203~ = 1.79, _p_ = 0.0747).
-
-### Substantial underrepresentation of honorees with East Asian names among US-affiliated scientists
-
-We applied our Wiki2019-LSTM model to US-affiliated authors and honorees and found similar pattern as when we considered all authors and honorees.
-We note that the US was not included in the training of the Wiki2019-LSTM model (see Methods).
-Specifically, the proportion of authors with Celtic/English names had decreased (Fig. {@fig:us_name_origin}A, left), particularly for papers published in _Bioinformatics_ and _BMC Bioinformatics_ (see [notebook](https://greenelab.github.io/iscb-diversity/14.us-name-origin.html#sup_fig_s6)).
-Meanwhile, the majority of honorees are predicted to have Celtic/English or European names (Fig. {@fig:us_name_origin}A, right).
-When we directly compared honoree composition with PubMed, we observed discrepancies between the two groups, namely a large overrepresentation of honorees with Celtic/English names (t~247~ = 6.46, _p_ < 10^-9^), a smaller overrepresentation of honorees with European names (t~250~ = 2.54, _p_ = 0.0059), and a substantial underrepresentation of honorees with East Asian names (t~329~ = -20.6, _p_ < 10^-16^) (Fig. {@fig:us_name_origin}B).
-No statistically significant difference was observed between the proportion of honorees and authors with names in other categories (see Table @tbl:example_names, t~252~ = -0.152, _p_ = 0.879).
-
-![(A) Estimated composition of name origin prediction over the years of
-  US-affiliated journal authors (left)),
-  and all US-affiliated Fellows and keynote speakers (right).
-  (B) For each region, the mean predicted probability of US-affiliated corresponding authors is shown as teal LOESS curve, and the mean probability and 95% confidence interval of the US-affiliated ISCB honoree predictions are shown as dark circles and vertical lines.
-  Compared to the name collection of US-affiliated Pubmed authors, US-affiliated honorees with Celtic/English and European names are overrepresented while US-affiliated honorees with East Asian names are underrepresented.
-  
-](https://raw.githubusercontent.com/greenelab/iscb-diversity/20d21504463c0487aa844be0ff236b3a51c70fa4/figs/us_name_origin.png){#fig:us_name_origin}
 
 ### Reception & Feedback
```

Comment:
I want to circle back to this a bit, but I will do it on a new branch.

Reply:
Okay!